### PR TITLE
005 README file updated

### PR DESCRIPTION
**Pull request by ReannaMcE**
The README file has been personalised to suit our project

### CODE REVIEW

**Review by joel-and — APPROVED** [review]

**Review by joel-and — CHANGES_REQUESTED** [review]
line 56- no longer including api test

**Review by joel-and — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -49,13 +49,40 @@ How to start the test application
...
     3. DB_HOST
     4. DB_NAME
-1. Run `mvn clean install` to build your application
-1. You can start application via:
+2. Run `mvn clean install` to build your application
+3. You can start application via:
     1. Terminal: `java -jar target/java-swagger-flyway-starter-org.kainos.ea.jar server config.yml`
     2. IDE: Edit run configuration -> Add `server` to program arguments -> Run
-1. To check that your application is running enter url `http://localhost:8080/api/test`
+4. To check that your application is running enter url `http://localhost:3000/jobRoles`
```
Comment (shaunganley) [inline README.md]: this is a frontend URL

File: README.md [hunk 2]
```diff
@@ -49,13 +49,40 @@ How to start the test application
...
+---
+The application allows the user to perform tasks such as checking job roles and adding and deleting job roles.
+
+
+
+Technology Stack
+---
+Backend: IntelliJ, Docker, AWS, MySQL
+Frontend: Bootstrap
+Testing: Selenium, WAVE & PA11Y
```
Comment (shaunganley) [inline README.md]: only need backend techs in this repo

**Review by shaunganley — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -49,13 +49,36 @@ How to start the test application
...
+Project Description
+---
+The application allows the user to perform tasks such as checking job roles and adding and deleting job roles.
+
+
+
+Technology Stack
+---
+Backend: IntelliJ, Docker, AWS, MySQL
+Testing: Selenium
```
Comment (shaunganley) [inline README.md]: selenium not used for testing backend

File: README.md [hunk 4]
```diff
@@ -49,13 +49,36 @@ How to start the test application
     2. DB_PASSWORD
     3. DB_HOST
     4. DB_NAME
-1. Run `mvn clean install` to build your application
-1. You can start application via:
+2. Run `mvn clean install` to build your application
+3. You can start application via:
     1. Terminal: `java -jar target/java-swagger-flyway-starter-org.kainos.ea.jar server config.yml`
     2. IDE: Edit run configuration -> Add `server` to program arguments -> Run
-1. To check that your application is running enter url `http://localhost:8080/api/test`
```
Comment (shaunganley) [inline README.md:56]: you should specify a valid backend URL here

**Review by shaunganley — CHANGES_REQUESTED** [review]

**Review by shaunganley — APPROVED** [review]

**Pull request merged**